src/hotspot/cpu/ppc/sharedRuntime_ppc.cpp

```diff
@@ -1980,34 +1980,38 @@ static void gen_continuation_yield(MacroAssembler* masm,
 
   Label L_pinned;
 
-  __ cmpdi(CCR0, R3_RET, 0);
+  __ cmpwi(CCR0, R3_RET, 0);
```

Comment:
Since I'm at it: the call above returns `int` so it is better to use a 32bit compare.

Reply:
@RealLucy please note this change.